collections/events.js

```diff
@@ -182,7 +182,7 @@ Events.updateGroups = function(eventId) {
 
 
 Meteor.methods({
-	saveEvent: function(eventId, changes, updateReplicas, sendNotifications) {
+	saveEvent: function(eventId, changes, updateReplicas, sendNotifications, addNotificationMessage) {
```

Comment:
Ah that's ok then.

Maybe just `comment` would work? But leave it if it works for you.